frontend/src/components/GamesBoard.js

```diff
@@ -0,0 +1,67 @@
+import GameTile from './GameTile';
+import styled from 'styled-components/macro';
+import ButtonTab from './ButtonTab';
+import ButtonBig from './ButtonBig';
+import { useState } from 'react';
+
+export default function GamesBoard({ games, selectGame }) {
+    const [gameToPlay, setGameToPlay] = useState();
+    const [noGameSelected, setNoGameSelected] = useState(false);
```

Comment:
I think this state is derived from `gameToPlay`. If I understand it correctly, `noGameselected` is truthy when `gameToPlay` is falsy.

Reply:
that is true, but I want to handle the error and respective conditional render of "Please select game" on the click event of the button. not using the boolean state would render the "PSG" part all the time until a game is chosen.

I would need be a different approach of guiding the user, which I might consider in future versions ^^